Testscripts/Linux/gpu-driver-install.sh

```diff
@@ -176,10 +176,8 @@ fi
 
 if [ -f /usr/libexec/platform-python ]; then
     ln -s /usr/libexec/platform-python /sbin/python
-    wget https://raw.githubusercontent.com/torvalds/linux/master/tools/hv/lsvmbus
-    chmod +x lsvmbus
-    mv lsvmbus /usr/sbin
 fi
+which lsvmbus || wget https://raw.githubusercontent.com/torvalds/linux/master/tools/hv/lsvmbus && chmod +x lsvmbus && mv lsvmbus /usr/sbin
```

Comment:
Is this right place? I think lsvmbus has dependency of python. Please help me understand

Reply:
yes, it needs python.